test/api-digester/Inputs/stdlib-stable-abi.json

```diff
@@ -16125,31 +16125,6 @@
         }
       ]
     },
-    {
```

Comment:
How come this was done instead of making the addition to the expected file like in #20948?

Reply:
Oh my apologies. When I cherry picked this my expected changes had a conflict because the expected was regenerated after I committed. I was also following a similar change made by Robert here: https://github.com/apple/swift/pull/21482 . I can remove this change in favor of adding the expected change tonight if that’s ok.